sklearn/utils/class_weight.py

```diff
@@ -59,3 +60,37 @@ def compute_class_weight(class_weight, classes, y):
                 weight[i] = class_weight[c]
 
     return weight
+
+
+def compute_sample_weight(class_weight, classes, y):
```

Comment:
The one in master has a different signature, I think, but it can probably be used.